src/Appium.Net/Appium/Service/AppiumCommandExecutor.cs

```diff
@@ -79,6 +80,8 @@ public Response Execute(Command commandToExecute)
             }
         }
 
+        public Task<Response> ExecuteAsync(Command commandToExecute) => Task.Run(() => this.Execute(commandToExecute));
```

Comment:
We should adapt a similar approach to the one the selenium team did.
Meaning, having the Excute method calling the ExcuteAsync and not the other way around.

Reply:
ok, no worries, will flip that